go/types.go

```diff
@@ -238,15 +244,15 @@ type SignedTreeHead struct {
 // 3.2, 4.1 and 4.2.
 type SignedCertificateTimestamp struct {
 	SCTVersion Version         // The version of the protocol to which the SCT conforms
-	LogID      SHA256Hash      // The SHA-256 hash of the (DER-encoded) public key for the Log
+	LogID      LogID           // The SHA-256 hash of the (DER-encoded) public key for the Log
 	Timestamp  uint64          // Timestamp (in ms since unix epoch) at which the SCT was issued
 	Extensions CTExtensions    // For future extensions to the protocol
 	Signature  DigitallySigned // The Log's signature for this SCT
 }
 
 func (s SignedCertificateTimestamp) String() string {
 	return fmt.Sprintf("{Version:%d LogId:%s Timestamp:%d Extensions:'%s' Signature:%v}", s.SCTVersion,
-		base64.StdEncoding.EncodeToString(s.LogID[:]),
+		hex.EncodeToString(s.LogID.KeyID[:]),
```

Comment:
I was generally shifting to hex because it makes much more sense for either ASN.1 encoded or for TLS encoded data -- it's possible to see the structure of the data in hex but not in base64.

However, that reasoning doesn't apply to non-encoded things like SHA256 hash values, and then I'm left with just a feeble consistency argument that is actually just a disguise for a personal dislike of base64.

So I'll ask around the team for preferences and probably come back and back out this part...